OrbitCore/Capture.cpp

```diff
@@ -35,6 +35,7 @@
 std::shared_ptr<Pdb> GPdbDbg;
 #endif
 
+Capture::State Capture::GState = Capture::State::kEmpty;
```

Comment:
Do we have a chance to not have this as a global?

Reply:
One day we'll move all these globals to some kind of `Capture` object but for now everything in `Capture` is a global.

Reply:
Okay, that is what I expected. Just wanted to double check.